todo_app/templates/index.html

```diff
@@ -18,7 +19,12 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         {% for item in items %}
-        <li>{{item.id}} - {{item.status}} - {{item.title}}</li>
+        <li>{{item.id}} - {{item.status}} - {{item.title}}
+          <div><form action='/move-item' method="POST">
+            <input type="text" id="cardId" name="cardId" value="{{item.id}}" hidden>
+            <input type="submit" id="submit" name="submit" value="move to Done">
+          </form></div><br>
```

Comment:
It would be good to only display the form for items which aren't already done, but not required